pkg/backend/httpstate/backend.go

```diff
@@ -229,6 +229,10 @@ func loginWithBrowser(ctx context.Context, d diag.Sink, cloudURL string, opts di
 	return New(d, cloudURL)
 }
 
+func SetDefaultOrg(url string, orgName string) error {
+	return workspace.SetBackendConfigDefaultOrg(url, orgName)
```

Comment:
Should this verify that the org is a real org the user has access to?  (Or at least warn if it is not?)

I'm not certain either way.  It would be useful feedback in case of typos, but may preclude using this command before creating an org?